src/lib_c/x86_64-windows-msvc/c/processthreadsapi.cr

```diff
@@ -50,6 +50,7 @@ lib LibC
                      bInheritHandles : BOOL, dwCreationFlags : DWORD,
                      lpEnvironment : Void*, lpCurrentDirectory : LPWSTR,
                      lpStartupInfo : STARTUPINFOW*, lpProcessInformation : PROCESS_INFORMATION*) : BOOL
+  fun SetThreadDescription(hThread : HANDLE, lpThreadDescription : LPWSTR) : BOOL
```

Comment:
Currently we do this for Win32 APIs available only in newer OS versions, and the caller must check `{{ LibC.has_method?(:SetThreadDescription) }}`:

```suggestion
  {% if LibC::WIN32_WINNT >= LibC::WIN32_WINNT_WIN10 %}
    fun SetThreadDescription(hThread : HANDLE, lpThreadDescription : LPWSTR) : LONG
  {% end %}
```

Also note `LONG` instead of `BOOL`, as the original return type is an `HRESULT`